io.github.Fontforge.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.Fontforge
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+rename-icon: fontforge
+rename-desktop-file: fontforge.desktop
+
+command: fontforge
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  # The file picker does not use portals :(
+  - --filesystem=host
+
+cleanup:
+    - "*.a"
+    - "*.la"
+    - "*.h"
+
+modules:
+  - shared-modules/python2.7/python-2.7.15.json
+  - name: pango
+    sources:
```

Comment:
Pango is in the runtime.

Reply:
I get this error without pango:

```
checking for PANGO... no
configure: error: Package requirements (pango >= 1.10 pangoxft) were not met:

Package 'pangoxft', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
```

so something's weird there. But including pango makes it work.

Reply:
Hmm, `pangoxft` is the missing one. Maybe it shouldn't be missing and reported here: https://gitlab.com/freedesktop-sdk/freedesktop-sdk

Reply:
Well now that I think about it nothing should be using `xft` in 2018. So maybe this app shouldn't require it.

Reply:
Could you add a comment saying that this is only for `pangoxft` though?